### PR TITLE
Feature/checkbox from typeahead

### PR DESCRIPTION
**Pull request by noahmanger**
Makes use of the new filter typeahead from https://github.com/18F/fec-style/pull/40

Creates a new macro for these fields.

### CODE REVIEW

File: static/js/init.js [hunk 1]
```diff
@@ -37,6 +38,16 @@ $('.js-dropdown').each(function() {
   new dropdown.Dropdown(this);
 })
 
+$('.js-typeahead-filter').each(function() {
+    var dataKey = $(this).attr('data-dataset');
+    var datasetMap = {
+        candidates: typeahead.datasets.candidates,
+        committees: typeahead.datasets.committees
+    }
+    var dataset = datasetMap[dataKey];
```
Comment (jmcarp) [inline static/js/init.js]: Do we need the `datasetMap` variable? How about `var dataset = typeahead.datasets[dataKey];`?

**Comment by jmcarp** [comment]
I was looking at this for a bit and realized we're going to have to handle this a little differently for performance reasons. For the big tables (receipts and disbursements), filtering on multiple committee IDs turns out to be prohibitively expensive. I think we're going to want to stick with a single filter and leave out the checkboxes. I'm going to do some more research and see if I can find a way to make multiple committees performant, but until then, let's hold off on merging.

**Comment by noahmanger** [comment]
Ah, good looking out. That's fine.

Even if we can't do multiple committees it would be great to have it set up so if there's a committee id applied on page load, it shows up in the filters. (Currently the committee name filter stays empty).

**Comment by noahmanger** [comment]
Closing this in the mean time.

**Pull request closed without merging**